ngrinder-controller/src/main/java/org/ngrinder/perftest/service/PerfTestService.java

```diff
@@ -891,6 +891,7 @@ String getProperSizedStatusString(Map<String, SystemDataModel> agentStatusMap) {
 	public ConsoleProperties createConsoleProperties(PerfTest perfTest) {
 		ConsoleProperties consoleProperties = ConsolePropertiesFactory.createEmptyConsoleProperties();
 		try {
+			removePerftestResultDistributedFolderIfExist(perfTest);

```

Comment:
I think this should not be called here..
it has the different behavior from createConsoleProperties()